scripts/generate/templates/PRODUCT_SUBSCRIPTION/react.template.js

```diff
@@ -1,3 +1,4 @@
+/* eslint react/prop-types: 0 */
```

Comment:
I would actually add this right into the [eslint config](https://github.com/Shopify/argo-admin-template/blob/master/.eslintrc.js). The prettierrc and eslintrc gets blown away during the clean up step of the generate process, so having this here wouldn't be useful after the template is setup.